config/routes.rb

```diff
@@ -65,6 +65,7 @@
     end
   end
 
+
```

Comment:
Minor: These white space changes can be removed from this pull request.

Reply:
Done